app/dev-app-update.yml

```diff
@@ -0,0 +1,3 @@
+owner: tsah20
```

Comment:
I see you added this file to the `.gitignore` 

Do they recommend to not commit it? 

If it's used for development, then maybe we should create `app/dev-app-update.yml.example` (so we don't need to google the format when we wanna use it for development ). What do you think?

Reply:
Yes, It should be added in .gitignore. If we want to use updater in a dev mode,we need to add this file manually.
Yes we can add app/dev-app-update.yml.example for a reference.

Reply:
Ok then ;]